documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -0,0 +1,252 @@
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+spec:
+  kafka:
+    # ...
+    listeners: <1>
+      external:
+        type: _LISTENER-TYPE_ <2>
+        tls: true <3>
+        authentication:
+          type: tls <4>
+        configuration:
+          preferredAddressType: InternalDNS <5>
+        networkPolicyPeers: <6>
+          - podSelector:
+              matchLabels:
+                app: kafka-consumer
+          - namespaceSelector:
+              matchLabels:
+                project: my-project
+        overrides: <7>
+          # ...
+      authorization: <8>
+        type: simple
+        superUsers:
+          - super-user-name <9>
+  # ...
+----
+<1> Configuration options for enabling external listeners are described in the link:{BookURLUsing}#type-KafkaListeners-reference[Kafka listeners schema reference^]
+<2> External listener type specified as `route`, `loadbalancer`, `nodeport` or `ingress`.
+<3> Enables TLS encryption on the listener. Not required for `route` listeners.
+<4> Authentication specified as `tls`.
+<5> (Optional, for `nodeport` listeners only) Configuration to link:{BookURLUsing}#con-kafka-broker-external-listeners-nodeports-{context}[specify a preference for the first address type used by Strimzi as the node address].
+<6> (Optional) By default, Strimzi grants access to listeners to all applications and namespaces. The `networkPolicyPeers` property restricts access to specific pods and namespaces.
+<7> (Optional, but not applicable to `ingress` listeners) Overrides customize the bootstrap and broker addresses advertised to clients.
+Strimzi automatically determines the addresses to advertise to clients.
+The addresses are automatically assigned by Kubernetes.
+Use overrides if the infrastructure on which you are running Strimzi does not provide the right address.
+Validation is not performed on overrides.
+The override configuration differs according to the external listener type,
+so you can override hosts for `route`, DNS names or IP addresses for `loadbalancer`, and node ports (shown) for `nodeport`.
+Refer to the link:{BookURLUsing}#type-KafkaListeners-reference[Kafka listeners schema reference^] for more information on external listener `overrides`.
+<8> Authorization link:{BookURLUsing}#ref-kafka-authorization-{context}[enables `simple` authorization on the Kafka broker using the `SimpleAclAuthorizer` Kafka plugin].
+<9> (Optional) Super users can access all brokers regardless of any access restrictions defined in ACLs.
+
+. Create or update the `Kafka` resource.
++
+[source,shell,subs=+quotes]
+oc apply -f _KAFKA-CONFIG-FILE_
++
+The Kafka cluster is configured with a Kafka broker listener using TLS authentication.
++
+A service is created for each Kafka broker pod.
++
+An additional service is created to serve as the _bootstrap address_ for connection to the Kafka cluster.
++
+The cluster (CA) to verify the identity of the kafka brokers is also created with the same name as the `Kafka` resource.
```

Comment:
Is (CA) correct here?

Reply:
Not quite. I've changed to _cluster CA certificate_